internal/users/idlimits.go

```diff
@@ -1,20 +1,8 @@
 // Code generated by "idlimitsgenerator"; DO NOT EDIT.
 package users
 
-// SystemdSystemUidMax is the Systemd configured Value for system_uid_max.
-const SystemdSystemUidMax uint32 = 999
-
-// SystemdSystemGidMax is the Systemd configured Value for system_gid_max.
-const SystemdSystemGidMax uint32 = 999
-
-// SystemdDynamicUidMin is the Systemd configured Value for dynamic_uid_min.
-const SystemdDynamicUidMin uint32 = 61184
-
-// SystemdDynamicUidMax is the Systemd configured Value for dynamic_uid_max.
-const SystemdDynamicUidMax uint32 = 65519
-
-// SystemdContainerUidBaseMin is the Systemd configured Value for container_uid_base_min.
-const SystemdContainerUidBaseMin uint32 = 524288
-
-// SystemdContainerUidBaseMax is the Systemd configured Value for container_uid_base_max.
-const SystemdContainerUidBaseMax uint32 = 1878982656
```

Comment:
Don't remember if you mentioned already, but what was the rationale for not excluding these?

Reply:
it's only used by systemd-nspawn, which is not installed by default on Ubuntu and is no different than LXD or other container technologies, so I don't see why we should treat it specially